parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -201,6 +201,23 @@ installImg() {
     retrycmd_get_executable 120 5 $img_filepath "https://acs-mirror.azureedge.net/img/img-linux-amd64-v0.5.6" ls || exit $ERR_IMG_DOWNLOAD_TIMEOUT
 }
 
+extractKubeBinaries() {
```

Comment:
I don't quite like the global var reference, can you let the caller pass in the k8s version explicitly to this method? Being explicit is always better coding practice IMO